courses/models.py

```diff
@@ -69,17 +69,27 @@ def __str__(self):
     class Meta:
         unique_together = ('program', 'position_in_program',)
 
-    def get_next_run(self):
+    def get_first_unexpired_run(self, course_run_to_exclude=None):
         """
-        Gets the next run associated with this course.
+        Gets the soonest unexpired run associated with this course.
         Note that it could be current as long as the enrollment window
         has not closed.
+
+        Args:
+            course_run_to_exclude (CourseRun): A CourseRun to exclude
+                from the query
+
+        Returns (CourseRun or None): An unexpired course run
```

Comment:
The syntax here should be `Returns: CourseRun or None: An unexpired course run`, broken into separate lines if you prefer: http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#docstrings